lib/Echidna/Transaction.hs

```diff
@@ -30,11 +32,14 @@ import EVM hiding (value)
 import EVM.ABI (abiCalldata, abiValueType, AbiValue(..), AbiType(..))
 import EVM.Concrete (Word(..), w256)
 import EVM.Types (Addr)
+import System.Directory hiding (listDirectory, withCurrentDirectory)
```

Comment:
maybe we can use normal qualified imports like everything else above this

Reply:
Fixed.